meson_options.txt

```diff
@@ -1,4 +1,11 @@
 option('portal-tests', type: 'array', choices: ['gtk3', 'gtk4', 'qt'], value: [],
   description : 'A list of portal tests to build')
+
+option('autotests', type: 'array', choices: ['qt'], value: [],
+description : 'A list of portal unit tests to build')
```

Comment:
I'd rather have this option as a boolean, and inside `tests/meson.build` you`d add something like:

```
if get_option('qt5')
  subdir('qt5')
endif
```